scripts/release_archive.py

```
@@ -0,0 +1,626 @@
+        return windows_wrapper
+    return _resolve_cargo_program_and_args(
+        stripped_cargo,
+        _split_cargo_command(stripped_cargo),
+    )
+
+
+def _normalise_cargo_command(cargo: str) -> str:
+    """Return a stripped, non-empty Cargo command string."""
+    stripped_cargo = cargo.strip()
+    if not stripped_cargo:
+        raise SystemExit("cargo executable cannot be empty")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)